app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable,
+         :recoverable,
+         :registerable,
+         :rememberable,
+         :validatable,
```

Comment:
The trailing comma after `:validatable` is going to raise a syntax error 😄

Reply:
Fixed!